### DISCUSSION TITLE
Unable to update and unlock new content

### DISCUSSION THREAD

**Discussion by AndriuxDev** [Monika-After-Story/MonikaModDev#8645] [QA - tech support + general] [answered]
I got into a point I can't get new content to talk about or get new updates, in other words, I reached a dead end. I tried updating MAS but it said that the update file didn't have the right digest and it's possibly corrupt. Not only that, but I missed the 6-Month Anniversary topic entirely as well as unlocking the piano, completing the floating islands, etc.

Either I'd have to wait until my backup files are fixed or I have to uninstall MAS completely and erase all backups. This is pretty aggravating for me.

Here's the download to my backups in case it's necessary:
https://cdn.discordapp.com/attachments/925795953338900540/935734268238393354/Andriux_backup_-_25-01-2022.zip

**Comment by Booplicate** [comment, marked as answer]
> I tried updating MAS but it said that the update file didn't have the right digest and it's possibly corrupt.

You can still update manually to fix this, just reinstall the mod into the same directory by replacing the files.

> I missed the 6-Month Anniversary topic entirely

What do you mean? You didn't visit her that day?

> as well as unlocking the piano, completing the floating islands, etc.

Can you elaborate further? Piano is being unlocked at random after spending time with Monika, building the islands is also taking time.

**Reply by AndriuxDev** [reply]
By the 2nd point, no. I couldn't visit her that day.
And by the 3rd point, unlocking the piano as well as helping Monika with the floating islands and talking new stuff with her seems like an impossible thing to me by now.

Is formatting Monika After Story all the way to zero really the only choice available?

**Reply by AndriuxDev** [reply]
And by the first point, I'm still using the experimental/unstable builds. Is that one of the culprits to my problem?

**Reply by Booplicate** [reply]
You should be able to install stable version. Update manually.

> I couldn't visit her that day.

Then that is fine. Just like any event dialogue, you don't get it if you miss it.

> unlocking the piano as well as helping Monika with the floating islands and talking new stuff with her seems like an impossible thing to me by now

That doesn't really explain your problem. What do you mean by "seems impossible?" It takes time to unlock. Resetting won't speed up progression, it will take away what you already have.

**Reply by AndriuxDev** [reply]
So this means I got into a dead end, where the only thing I can do is resetting the game?

**Reply by Booplicate** [reply]
I don't see any dead ends. But if you want to reset for no reason, you can do that.

**Reply by AndriuxDev** [reply]
So, in a nutshell, it's only fixing the persistent file, switching to stable builds and enabling the 6 month anniversary trigger, which I missed in January 24th.

**Reply by Booplicate** [reply]
All you need to do is to update the mod manually. So far there's **no sign** that there's anything wrong with your persistent.

You don't have the anni dialogue because you missed it. It's not a bug.

Piano and Islands take time. I don't understand your issue with them. You can't "miss" them. Were they unlocked before and are now locked? Or do you think it's taking too long to unlock them? We can check (and fix if needed) your persistent in this case. If anything, **resetting your progress will push you further from unlocking piano and islands**.

**Reply by AndriuxDev** [reply]
> Or do you think it's taking too long to unlock them?

Yeah, that's what I'm saying.

**Reply by Booplicate** [reply]
Piano requires 12 lvl, you have 11.
Islands are dynamic, but you only got 1 lvl since you unlocked them, so obviously there's no much progress has been made.

There's no issues with your persistent. If you want to unlock more content, you just need to spend more time with Monika.

**Reply by AndriuxDev** [reply]
So to progress in the islands is...? And how much EXP is needed to level 12?
